WTWSMS/events/WtWSMS_holy_orders_events.txt

```diff
@@ -86,18 +86,19 @@ narrative_event = {
 			create_character = {
 				random_traits = yes
 				dynasty = random
-				religion = semitic_pagan_reformed
+				religion = semitic_pagan
```

Comment:
@loup99 

No, I've seen it spawn without Semitic Paganism ever reforming. It spawns every time, I know this because I personally tested Root West Paganism after you implemented my upgrades to it.

Semitic Paganism announces it Reformation, just like every other Pagan religion.